webviewer/src/pages/Account/Account.js

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+import { Page } from 'components/_shared/Page';
+import { ContactUs } from 'components/ContactUs';
+import { Sidebar } from './components';
+import { Wrapper } from './Account.styles';
+import { PageContent } from './components';
+const TABS = [
+  { name: 'Profile', icon: 'Settings' },
+  { name: 'Transactions', icon: 'Document' }
+];
+
+export const Account = ({ ...props }) => {
```

Comment:
![image](https://user-images.githubusercontent.com/23707448/224642876-e6369a9b-4d90-426b-8d7e-f36b2ed6d463.png)
is such padding value was into mockups? looks not good, maybe will be better to make them equal?

Reply:
yes it's like in design, padding is different